ui/src/components/easter/hero/PopUp.js

```diff
@@ -0,0 +1,305 @@
+              textStyle="Quicksand_bold"
+              color="#003984"
+            >
+              The online Easter Celebration stream is now
+              <Text
+                fontSize="40"
+                textStyle="Quicksand_bolder"
+                textDecorationLine="underline"
+                background="none"
+                paddingLeft="45%"
+              >
+                LIVE
+              </Text>
+            </Text>
+            <CloseButton onClick={closeModal} size="sm" />
+          </Stack>
+          <Stack direction="row"></Stack>
+          <Button
+            alignSelf="center"
+            height="auto"
+            as={Link}
+            p="2"
+            background="#537893"
+            color="#ffffff"
+            width="30%"
+            backdropFilter="blur(6px)"
+            borderRadius="10px"
+            href="/online"
+          >
+            <Text fontSize="20" textStyle="Quicksand_bold">
+              Check it out!
+            </Text>
+          </Button>
+        </Stack>
+      </Box>
+    );
+  };
+
+  const MobileViewSunday = () => {
+    return (
+      <Box
+        position="absolute"
+        right="10px"
+        minH="fit"
+        height="fit-content"
+        width="90vw"
+        background="#FFFFFF"
+        display={{ base: displayModal, md: 'none' }}
+        p="6"
+        zIndex="sticky"
+        boxShadow="0px 3px 4px 2px rgba(0, 0, 0, 0.3)"
+        borderRadius="20px"
+      >
+        <Stack direction="column">
+          <Stack direction="column" spacing={0}>
+            <CloseButton onClick={closeModal} size="lg" alignSelf="flex-end" />
+            <Text
+              fontSize="25"
+              marginTop="10px"
+              textStyle="Quicksand_bold"
+              color="#003984"
+              justifyContent={'center'}
+              textAlign="center"
+            >
+              The online Easter Celebration stream is now{' '}
+              <Text
+                fontSize="40"
+                //paddingLeft = "35%"
+                textStyle="Quicksand_bolder"
+                textDecorationLine="underline"
+                display="inline"
+                color="#003984"
+              >
+                LIVE
+              </Text>
+            </Text>
+          </Stack>
+          <Center>
+            <Button
+              alignSelf="center"
+              height="auto"
+              as={Link}
+              p="2"
+              background="#537893"
+              top="2"
+              color="#ffffff"
+              width="80%"
+              backdropFilter="blur(6px)"
+              borderRadius="10px"
+              href="/online"
+            >
+              <Text fontSize="20" textStyle="Quicksand_bold">
+                Check it out!
+              </Text>
+            </Button>
+          </Center>
+        </Stack>
+      </Box>
+    );
+  };
+  // let currDate = new Date('17 Apr 2022 02:30:00 UTC');
+  let currDate = new Date();
```

Comment:
minor, use const if variable is never reassigned

Reply:
also minor, but please use luxon next time. helps keep things consistent thruout our whole code